Tests/MSFT_xDhcpServerAuthorization.Tests.ps1

```diff
@@ -0,0 +1,134 @@
+[CmdletBinding()]
+param()
+
+if (!$PSScriptRoot) # $PSScriptRoot is not defined in 2.0
```

Comment:
I'm not sure this code is strictly necessary because DSC is only supported on PS 4.0 - so it should only be tested on PS4.0 and above.